blank-canvas-blocks/experimental-theme.json

```diff
@@ -148,9 +148,23 @@
 					}
 				},
 				"quote": {
+					"typography": {
+						"textAlign": "left"
+					},
 					"citation": {
 						"typography": {
-							"fontSize": "var(--wp--preset--font-size--small)",
+							"fontSize": "0.8em",
```

Comment:
IMO we should be using a size from the theme rather than a new size.

Reply:
I liked that by using `em` units in this way the citation tracks against the size of the quote.  And particularly since the quote citation font size isn't something the user has control over.  Ultimately I see that as a theme design decision though and I'm mostly impartial as to what that "default" decision is, only that it is one that CAN be expressed.

Reply:
Makes sense. My thought was that one important thing that keeps designs from looking messy is having a small set of font sizes, so my concern is that this would add another. Maybe @kjellr can decide?

Reply:
👍 Makes sense to use a standardized size.  I don't know why I've been on such an 'em' kick lately...
Replaced both with `--wp--preset--font-size--tiny` and there was no visible change.